lib/common/options.c

```diff
@@ -448,8 +448,7 @@ static pcmk__cluster_option_t cluster_options[] = {
         PCMK__OPT_REMOVE_AFTER_STOP, NULL, "boolean", NULL,
         PCMK_VALUE_FALSE, pcmk__valid_boolean, pcmk__opt_deprecated,
         pcmk__opt_context_schedulerd,
-        N_("*** Deprecated *** "
-            "Whether to remove stopped resources from the executor"),
+        N_("Whether to remove stopped resources from the executor"),
```

Comment:
The deprecated and advanced flags should also cause the `*** Deprecated ***`/`*** Advanced use only ***` labels to be shown in text output. In XML output, we should use OCF's `<deprecated/>` capability, and I guess a custom `advanced="true"` XML attribute

Reply:
That's reasonable. I omitted them to avoid clutter since (1) as of the most recent push we had stopped displaying deprecated options and (2) the user indicated they were fine with seeing advanced options since they specified `--all`.

We're bringing deprecated back though.

Reply:
I think if someone runs --all they still benefit from knowing which options merit more caution

Reply:
You're right

Reply:
Do you think these labels should precede the short description, or be their own bullet points before or after the long description?

Reply:
I lean toward first bullet point, above long desc...

Reply:
Some possibilities:

    * remove-after-stop: Whether to remove stopped resources from the executor
      * DEPRECATED
      * Values other than default are poorly tested and potentially dangerous. This option will be removed in a future release.
      * Allowed values: true, false (default)
    
    * remove-after-stop (DEPRECATED): Whether to remove stopped resources from the executor
      * Values other than default are poorly tested and potentially dangerous. This option will be removed in a future release.
      * Allowed values: true, false (default)
    
    * remove-after-stop: Whether to remove stopped resources from the executor (DEPRECATED)
      * Values other than default are poorly tested and potentially dangerous. This option will be removed in a future release.
      * Allowed values: true, false (default)
    
    * remove-after-stop: Whether to remove stopped resources from the executor
      * Values other than default are poorly tested and potentially dangerous. This option will be removed in a future release. (DEPRECATED)
      * Allowed values: true, false (default)

Alternatively, we could show all non-advanced, non-default options first, then advanced options (with a heading), then deprecated options (with a heading). I kind of like that idea.